### PR TITLE
mark-read-reading

### PR DESCRIPTION
**Pull request by anhkt**

### CODE REVIEW

File: app/controllers/book_marks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    redirect_to @book
+  end
+
+  private
+  def book_marks_params
+    params.require(:book_mark).permit :read_status, :user_id, :book_id
+  end
+
+  def load_mark
+    @book_mark = BookMark.find_by id: params[:id]
```
Comment (yeunuthongminh) [inline app/controllers/book_marks_controller.rb]: co khi nao khong tim thay bookmark khong?

File: app/controllers/books_controller.rb [hunk 2]
```diff
@@ -4,6 +4,9 @@ class BooksController < ApplicationController
   def show
     @reviews = @book.review_rates.desc.paginate page: params[:page],
       per_page: Settings.per_page
+    @book = Book.find_by id: params[:id]
```
Comment (yeunuthongminh) [inline app/controllers/books_controller.rb]: neu khong tim thay thi sao?

File: app/models/activity.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 class Activity < ApplicationRecord
   belongs_to :user
+  has_many :likes
```
Comment (yeunuthongminh) [inline app/models/activity.rb]: co dependent::destroy k nhi?

File: app/models/review_rate.rb [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-  enum read_status: { Reading: 0, Read: 1, UnRead: 2}
+  enum read_status: {reading: 0, read: 1, unread: 2}
 
   belongs_to :user
   belongs_to :book
 
   has_many :comments, dependent: :destroy
   
-  validates :content, presence: true, length: {minimum: 50, maximum: 5000}
+  validates :content, presence: true, length: {minimum: 3, maximum: 5000}
```
Comment (yeunuthongminh) [inline app/models/review_rate.rb]: Setting

File: app/controllers/book_marks_controller.rb (hunk 1 above)
Comment (anhkt) [inline app/controllers/book_marks_controller.rb]: Co, co the bookmark se khong tim thay.

Comment (yeunuthongminh) [inline app/controllers/book_marks_controller.rb]: the sao khong bat truong hop nil?

**Comment by yeunuthongminh** [comment]
OK

**Pull request merged**